pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -118,6 +118,11 @@ def test_moral_graph_with_edge_present_over_parents(self):
             self.assertTrue(edge in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')] or
                             (edge[1], edge[0]) in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')])
 
+    
+    def test_local_independencies(self):
+        from pgmpy.independencies import Independencies
```

Comment:
@ankurankan Thanks. 
Even I was wondering the same but I saw a few import statements within the test functions. [[1]](https://github.com/pgmpy/pgmpy/blob/dev/pgmpy/tests/test_models/test_BayesianModel.py#L183) 
So I thought that it was fine to do so.
But I have rectified that now.